### DISCUSSION TITLE
Qtable keeps selection when :selection is turnet to 'none'

### DISCUSSION THREAD

**Discussion by RamonDonadeu** [quasarframework/quasar#15401] [General - Components / Directives / etc]
I have a QTable configured with a PINIA store, that, when a variable called, 'Selected' changes to 'single' or 'multiple', the qtable displays the checkbox on each row to select whatever. 

Last time i remember using this, when i turn back 'Selected' into 'none', the selected columns where clean without doing nothing special, but today I found out this doesn't work, and the rows keep selected. Is this a Bug? Or the feature just changed? Or I was the one who changed something, (I didn't found anytihing different in my code, so I don't think I caused the problem)

**Comment by MilosPaunovic** [comment]
Create a Codepen reproduction of the issue.

**Reply by rdonadeu** [reply]
https://codepen.io/RamonDonadeu/pen/KKBYeRz?editors=1011

Here it is, hopefully I did it right

**Reply by rdonadeu** [reply]
Anyway, I found a solution for my case. 
I set my v-model to my storage.selected.
Before, I had the normal v-model:selected="selected", and then 
    @selection="updateActionStore"
which updated storage.selected, but I'm pretty sure that when I turned selection to 'none', v-model:selected="selected" was emptied

**Reply by MilosPaunovic** [reply]
Often making a reproduction leads to finding a solution, so it's always a good starting point in research.
